cmd/fleetctl/package.go

```diff
@@ -50,12 +50,6 @@ func packageCommand() *cli.Command {
 				Value:       "com.fleetdm.orbit",
 				Destination: &opt.Identifier,
 			},
-			&cli.StringFlag{
-				Name:        "version",
-				Usage:       "Version for package product",
-				Value:       "0.0.5",
-				Destination: &opt.Version,
-			},
```

Comment:
The PR looks reasonable, but the CLI change confuses me because I was expecting to see an update to tests. Is there a ticket for this? Should we add a changes file for orbit?

Reply:
With the changes in this PR, the `fleetctl package` always sets the package version to the latest orbit version as reported by the TUF (also, we were only using such `opt.Version` option on linux packaging only).

Reply:
Makes sense to me. I'll defer to @zwass for the final call though.

Reply:
Oh, according to the CI failure, opt.Version is used in the Windows metadata template. I'll readd.